File: fix_table.js
const fs = require('fs');
['frontend/src/pages/admin/MembershipQueue.tsx', 'frontend/src/pages/admin/MembersList.tsx', 'frontend/src/pages/admin/LoanQueue.tsx', 'frontend/src/pages/admin/LoansList.tsx'].forEach(p => {
  let c = fs.readFileSync(p, 'utf8');
  c = c.replace(/<table className=\"w-full text-sm text-left border-collapse\"(?!\s*>)/g, '<table className="w-full text-sm text-left border-collapse">');
  fs.writeFileSync(p, c);
});
console.log('Fixed >');

File: fix_table.py
import re

files = [
    'frontend/src/pages/admin/MembershipQueue.tsx',
    'frontend/src/pages/admin/MembersList.tsx',
    'frontend/src/pages/admin/LoanQueue.tsx',
    'frontend/src/pages/admin/LoansList.tsx'
]

for fp in files:
    with open(fp, 'r', encoding='utf-8') as f:
        c = f.read()
    
    # Just in case there is no newline
    c = re.sub(r'<table className="w-full text-sm text-left border-collapse"(?!\s*>)', '<table className="w-full text-sm text-left border-collapse">', c)
    
    with open(fp, 'w', encoding='utf-8') as f:
        f.write(c)

print("Fixed tags")
